lib/disc.rb

```diff
@@ -21,7 +27,7 @@ def self.disque=(disque)
   end
 
   def self.disque_timeout
-    @disque_timeout ||= 100
+    @disque_timeout ||= Integer(ENV.fetch('DISQUE_TIMEOUT', 2000))
```

Comment:
Not entirely sure why `Disc.disque_timeout` had a default different from `DISQUE_TIMEOUT`. Was this on purpose?